libretroshare/src/tor/TorControl.cpp

```diff
@@ -68,6 +68,11 @@ TorControl::TorControl()
     : mControlPort(0),mSocksPort(0),mStatus(NotConnected), mTorStatus(TorOffline),mHasOwnership(false)
 {
     mSocket = new TorControlSocket(this);
+    mControlPort = 0;
```

Comment:
Looks like we both fixed that at the same time ;-)

Reply:
Yes, there were some strange behaviors without the initialization.